rasa/utils/tensorflow/models.py

```diff
@@ -47,6 +48,25 @@
 logger = logging.getLogger(__name__)
 
 
+def _merge_batch_outputs(
+    all_outputs: Dict[Text, Union[np.ndarray, Dict[Text, np.ndarray]]],
+    batch_output: Dict[Text, Union[np.ndarray, Dict[Text, np.ndarray]]],
+) -> Dict[Text, Union[np.ndarray, Dict[Text, np.ndarray]]]:
+    if not all_outputs:
```

Comment:
Could you add a simple docstring? I'm hesitant to agree that the function is ["very short" and also "obvious"](https://google.github.io/styleguide/pyguide.html#383-functions-and-methods).